packages/algob/src/lib/script-checkpoints.ts

```diff
@@ -195,13 +195,15 @@ export function persistCheckpoint (scriptName: string, checkpoint: Checkpoints):
 /**
  * Register checkpoints for ASA and SSC
  * @param deployer Deployer object
- * @param txns transaction array
+ * @param execParams user transaction(s) input
+ * @param txns SDK transaction array
  * @param txIdxMap transaction map: to match transaction order
  * transaction index mapped to [asset name, asset definition]
  */
 /* eslint-disable sonarjs/cognitive-complexity */
 export async function registerCheckpoints (
   deployer: Deployer,
+  execParams: wtypes.ExecParams[],
```

Comment:
let's document this in checkpoint (breaking change)

Reply:
This is internal, not exposed to user